docs/backups.md

```diff
@@ -19,11 +19,13 @@ cluster using the following remote backup storages:
 
 ## Backup types
 
-<a name="physical"></a> The Operator can do either *logical* or *physical* backups.
+<a name="physical"></a> The Operator can do the following backups.
 
-* *Logical backup* means querying the Percona Server for MongoDB for the database data and writing the retrieved data to the remote backup storage.
+* *Full logical backup* means querying Percona Server for MongoDB for the database data and writing the retrieved data to the remote backup storage. 
 
-* *Physical backup* means copying physical files from the Percona Server for MongoDB `dbPath` data directory to the remote backup storage.
+* *Full physical backup* means copying physical files from the Percona Server for MongoDB `dbPath` data directory to the remote backup storage.
+
+* *Physical incremental* means copying only the data that was changed after the previous backup was taken. Using this type of backups speeds up backup / restore performance, lowers network load and storage consumption.
```

Comment:
I believe we need to add info re this pbm limitation for incremental backups:
`One consideration we need to keep in mind:
Percona Backup for MongoDB tracks the backup history only on the node where the base incremental backup was taken. This means that subsequent incremental backups must always be run on that very node. To make this happen, Percona Backup for MongoDB tries to schedule backups on that same node.

If the node with the base incremental backup is down or unavailable, you must start the incremental backup chain anew on another node.` 

This may be required after a restore.

Reply:
How can a user ensure this? And how can they learn that the node for backups has changed? 
 can I reword that not PBM but the Operator tries to schedule backups on that same node to ensure the backup history? 
 another question: what happens if the node is down for scheduled backups? Will the Operator run a new base automatically?